src/main/app/idam-api/IdamAPI.ts

```diff
@@ -74,6 +74,17 @@ export class IdamAPI {
       });
   }
 
+  public deleteSsoById(id: string) {
+    return this.userAxios
+      .delete('/api/v1/users/' + id + '/sso')
+      .catch(error => {
+        const errorMessage = 'Error removing user sso by ID from IDAM API';
```

Comment:
The error says remove sso by the method name is delete sso. They need to be made consistent

Reply:
fixed